client/gatsby-config.js

```diff
@@ -11,7 +11,7 @@ const curriculumIntroRoot = path.resolve(__dirname, './src/pages');
 module.exports = {
   siteMetadata: {
     title: 'freeCodeCamp',
-    siteUrl: 'https://www.freecodecamp.org'
+    siteUrl: config.siteUrl
```

Comment:
```suggestion
    siteUrl: config.homeLocation
```

We can re-use the `HOME_LOCATION` instead.